masterlist.yaml

```diff
@@ -18481,16 +18481,34 @@ plugins:
         condition: 'checksum("training spells.esp", DD4985A)'
   - name: 'Judgement of the Nine.esp'
     tag: [ Graphics ]
-  - name: 'AgarMoreVariedSpellEffects.esp'
-    tag: [ Sound ]
-  - name: 'cidnothumpMMversion.esp'
+
+  - name: '(AgarMoreVariedSpellEffects|AgarVariedSpellSounds_SupremeMagicka)\.esp'
+    url: [ 'https://www.nexusmods.com/oblivion/mods/14799' ]
     tag: [ Sound ]
+    msg:
+      - <<: *useOnlyOneX
+        subs: [ 'Varied Spell Sounds' ]
+        condition: 'many("(AgarMoreVariedSpellEffects|AgarVariedSpellSounds_SupremeMagicka)\.esp")'
+
```

Comment:
@Infernio Same as in https://github.com/loot/oblivion/pull/335#discussion_r611807143.

```suggestion
```